packages/babel-core/src/util.js

```diff
@@ -22,140 +15,3 @@ export function message(opts: Object, msg: string) {
 
   return `[BABEL] ${opts.filename || "unknown"}: ${msg}`;
 }
-
-/**
- * Test if a filename ends with a compilable extension.
- */
-
-export function canCompile(filename: string, altExts?: Array<string>): boolean {
```

Comment:
Curious for people's thoughts on this one. The current name is misleading since it's not really about what Babel _can_ compile so much as providing the standard file extensions for Babel. Even then, it seems to me like it should be up to the users of `babel-core` to decide what to run through it.

That said, I do somewhat see the benefit of having a standardized set of extensions. What we _could_ do is expose a `babel.standardExtensions`, but even then I'm not clear what we get from that vs tools deciding on their own. It's also something I'd be hesitant to change without a major version bump anyway, since who knows what people will be using in the wild.